tests/test_integration.py

```diff
@@ -124,7 +124,7 @@ def test_build(monkeypatch, project, args, call, tmp_path):
 
 def test_isolation(tmp_dir, package_test_flit, mocker):
     try:
-        import flit_core  # noqa: F401
+        import flit_core  # noqa: F401  # imported but unused
```

Comment:
Just curious, would `import flit_core as _` work?

Reply:
I guess the correct way to do this would be checking `importlib.util.find_spec("flit_core")` against `None`.

Reply:
> Just curious, would `import flit_core as _` work?

It does not.

Reply:
Could try adding `del flit_core` to the else-block, then.

Reply:
`importlib.util.find_spec("flit_core")` is the correct solution imo. It’s also faster if the module is large, since you don’t have to load it. Not that important for tests, but nice to do things the right way as a good example.

Reply:
And you wouldn’t need an else block for the del. It’s never used, so it could be in the try.

Reply:
You'd just need to reference it; `flit_core` on a new line would be sufficient.  I don't think we need to do any of those things however, silencing the warning where it does not apply is perfectly fine.

Reply:
@henryiii yep, using importlib is certainly better, but I was suggesting to use the fact that the else-block is already present, w/o much refactoring.
Also, referencing it in the try-section is a code smell because it should always have exactly one instruction.